block-storage/src/main/scala/coop/rchain/blockstorage/BlockDagFileStorage.scala

```diff
@@ -152,7 +152,7 @@ final class BlockDagFileStorage[F[_]: Concurrent: Sync: Log: BlockStore: RaiseIO
                      Option(children).pure[F]
                    case None =>
                      for {
-                       blockOpt <- BlockStore[F].get(blockHash)
+                       blockOpt <- none[BlockMessage].pure[F]
```

Comment:
If you do this then we won't be able to fetch from older checkpoints right?

Reply:
Temporarily, yes. Until I finish my work on RCHAIN-3019 (see my comment above). I need a way to look up block number by its hash here and I was relying on BlockStore before but I shouldn't.